developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,188 @@ title: "Mint an NFT with Aptos CLI"
 slug: "mint-nft-cli"
 ---
 
-# Mint an NFT
+# Mint an NFT with Aptos CLI
+
+This tutorial lets you use the Aptos CLI to mint NFTs in Aptos testnet so you can see how the process works and employ related functions in your code.
 
 ## Prerequisites
 
 This tutorial assumes you have:
 
-* a GitHub account
-* the GitHub CLI
-* the Aptos CLI (installed below)
+* a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+* the [GitHub CLI](https://cli.github.com/)
+* the [Aptos CLI](../../cli-tools/aptos-cli-tool/install-aptos-cli.md) (or you can run from [aptos-core](https://github.com/aptos-labs/aptos-core) source via `cargo run`)
 
-## Mint with the Aptos CLI
+## Role of accounts
 
-Now that you are starting to write smart contracts with Move, let's create our first NFT with the Aptos CLI.
+When you are minting an NFT, the NFT is tied to your [account](../../concepts/accounts.md). Aptos ensures no one else can alter your collection of NFTs. This is why a private key is required to obtain signer capabilities. When you submit a transaction, you sign the transaction.
 
-1. [Install the Aptos CLI](../../cli-tools/aptos-cli-tool/install-aptos-cli.md) and note its [many uses](../../cli-tools/aptos-cli-tool/use-aptos-cli.md) for later if you haven't experienced its goodness already.
+[Resource accounts](../resource-accounts.md) allow the delegation of signing transactions. You create a resource account to grant a signer capability that can be stored in a new resource on the same account and can sign transactions autonomously. The signer capability is protected as no one has access to the private key for the resource account.
+
+## Understand minting
+
+Review the [NFT Tutorial](https://github.com/aptos-labs/nft-tutorial/tree/main/tutorial) source code and accompanying READMEs.
+
+Explore the `mint_nft` function in [`minting.move`](https://github.com/aptos-labs/nft-tutorial/blob/main/sources/minting.move).
```

Comment:
I'm afraid that all of these need to change since we'll need to link to the more comprehensive tutorial...
really sorry about the confusion, I should have clarified sooner. 
I think what you have here is spot on, we'll probably just need to do some minor naming changes (e.g. `mint_nft` is `mint_event_pass` in the new tutorial). it's because some folks mentioned that the variable name is confusing, so I gave more verbose/explicit variable names [here](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/mint_nft).

Reply:
Thanks! Let me see what I can do. Please let's leave this comment open for tracking.

Reply:
These should all be updated now.